cmd/auth-provider-gcp/app/getcredentials.go

```diff
@@ -49,25 +49,31 @@ func NewGetCredentialsCommand() (*cobra.Command, error) {
 		RunE:  getCredentials,
 	}
 	defineFlags(cmd)
-	if err := validateFlags(); err != nil {
+	if err := validateFlags(authFlow); err != nil {
```

Comment:
We should avoid using a global variable for this. Where is this being set? It seems like this should be coming from the passed in flags?

Reply:
Right, done.